orttraining/orttraining/training_ops/cuda/math/softmax_grad_impl.cu

```diff
@@ -66,21 +169,21 @@ __global__ void softmax_warp_backward(output_t* gradInput, const input_t* grad,
   // This should have no impact on performance because the loops are unrolled anyway.
 
   // load data from global memory
-  acc_t grad_reg[WARP_BATCH][WARP_ITERATIONS];
-  acc_t output_reg[WARP_BATCH][WARP_ITERATIONS];
-  acc_t grad_output_reg[WARP_BATCH][WARP_ITERATIONS];
+  input_t grad_reg[WARP_BATCH][WARP_ITERATIONS];
+  input_t output_reg[WARP_BATCH][WARP_ITERATIONS];
```

Comment:
use output_t for output_reg?

Reply:
output_**reg** means the variable is stored in GPU register file, same nameing convention as softmax_warp_backward1

Reply:
OK， it is not output, it's from input. :)